src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -330,6 +339,10 @@ private void validate() {
 			this.databaseName = "maxwell";
 		}
 
+		if ( this.clientID == null ) {
+			this.clientID = "maxwell";
```

Comment:
would be better to make this unique so we can identify from which maxwell instance this is coming from? or we don't care that much?
```
An id string to pass to the server when making requests. The purpose of this is to be able to track the source of requests beyond just ip/port by allowing a logical application name to be included in server-side request logging.
```